tools/cli_install.php

```diff
@@ -156,6 +158,17 @@
                    WHERE `name` like 'full access from localhost'";
 $DB->query($apiClientQuery);
 
+if($dev) {
+   $entityConfig = new PluginFlyvemdmEntityConfig();
+   $entityConfig->getFromDBByCrit([
+      'entities_id' => '0',
+   ]);
+   $devquery = "UPDATE `glpi_plugin_flyvemdm_entityconfigs`
+               SET `download_url` = '" . PLUGIN_FLYVEMDM_AGENT_BETA_DOWNLOAD_URL . "'
+               WHERE `glpi_plugin_flyvemdm_entityconfigs`.`entities_id` = 0";
+   $DB->query($devquery);
```

Comment:
Why not update the entityConfig with the following code ? 
```php
   $entityConfig->update([
      'id'           => $entityConfig->getID(),
      'download_url' => PLUGIN_FLYVEMDM_AGENT_BETA_DOWNLOAD_URL
   ]);
```

(If you don't see any problem with that, I'm ready to commit to your pr, then merge.)

Reply:
You're right. This was a draft I forgot to drop when I commented this part of code... Removing